templates/plugin/python/module.conf.erb

```diff
@@ -1,7 +1,7 @@
   Import "<%= @module %>"
 
   <Module "<%= @module %>">
-  <% @config.sort.each do |key,value| -%>
-    <%= key -%> <%= value -%>
-  <% end %>
+  <%- @config.sort.each do |key,value| -%>
+    <%= key %> <%= value %>
+  <%- end -%>
```

Comment:
@deric when I replace `<%- end -%>` by `<% end %>` I get extra (not needed) new line before `</Module>` and wrong indentation in the `<Module>` block:

```
Notice: /Stage[main]/Collectd::Plugin::Python/Concat[/etc/collectd.d/python-config.conf]/File[/etc/collectd.d/python-config.conf]/content: 
--- /etc/collectd.d/python-config.conf  2015-06-30 09:45:40.752012515 +0200
+++ /tmp/puppet-file20150703-10516-16ym6qz-0    2015-07-03 08:50:02.977012923 +0200
@@ -8,19 +8,21 @@
 
   <Module "elasticsearch">
     Cluster opslogstash
-    Verbose false
-    Version 1.0
+      Verbose false
+      Version 1.0
+  
   </Module>
 
   Import "rabbitmq"
 
   <Module "rabbitmq">
     Host "localhost"
-    Password "ppp"
-    Port "15672"
-    PrefixVhost true
-    Realm "RabbitMQ Management"
-    Username "uuu"
+      Password "ppp"
+      Port "15672"
+      PrefixVhost true
+      Realm "RabbitMQ Management"
+      Username "uuu"
+  
   </Module>
 
 </Plugin>
```